### DISCUSSION TITLE
Max Farm Size

### DISCUSSION THREAD

**Discussion by scoville07** [madMAx43v3r/chia-gigahorse#264] [General]
Hello,

does anyone know if the max farm size, which is mentioned in the table, does mention compressed or uncompressed?

Best regards,
Christian

**Comment by scoville07** [comment]
@madMAx43v3r you probably know best, can you please enlight me.. ;)

**Comment by madMAx43v3r** [comment]
It's compressed size, aka physical size.

**Comment by DudSeb** [comment]
Hi Max, 

I would like to ask about max farm size. Currently Im testing max farm size on 2 x 4070 Ti on k32 c20 plots. With diff 100 the result is lower than with diff 1000. It s look like this for @256:

Diff 100: 0.476865 PiB
Diff 1000: 0.594171 PiB

I ve always thought that than higher difficulty the lower result should be. 

So, if I will set up a diffculty on the pool to 1000, did I'll be able to farm a size of 0.594171 PiB after filter change to 256 with this two 4070 Ti ?

**Comment by madMAx43v3r** [comment]
Yes the results are correct, higher diff = less compute = higher max farm size.

**Reply by DudSeb** [reply]
Thank you very much :)

**Comment by scoville07** [comment]
@madMAx43v3r oh wow, a new file format, i just finished replotting 1,8PB in C18...🤪

Would you mind to give a max farm size for rtx 4 series as well? Most people can't afford a RTX A5000 but maybe a RTX 4070Ti Super? I have RTX3060 currently but would plan on replacing them accordingly.

Regards and thanks for your hard work!

**Reply by DudSeb** [reply]
Hi there, i ve got 2 x 4070ti super i can provide the results, just tell me what difficulty you need and which compression level; :) But i can do this next week, now i'm far away from my farm ;(

**Reply by scoville07** [reply]
Cool, thanks for your offer. Currently in pool.space I run a difficulty of approx. 1000. Compression righ now C18, but with 3.0 format I would probably aim for C31.

Thanks!

**Reply by madMAx43v3r** [reply]
I don't have any 4000 series to test with...

**Comment by trxman** [comment]
Which difficulty should be set for testing max size of farm for regular pool farming?

**Reply by DudSeb** [reply]
Well basicly as far as i observed, it s should working like 1 PB farm = difficulty 100, 10 PB farm = difficulty 1000. But the less difficulty you need the more compute power. So i think if you have 1,8 Pb your diffculty should be around 180-200. It s working in my case on c20 plots.

**Reply by madMAx43v3r** [reply]
The new C30 to C33 need much higher diff to be effective, see the chart: 20k to 100k

Otherwise max farm size is only half.

**Reply by trxman** [reply]
Will increasing the difficulty / lowering amount of partials negatively influence farm size estimation by my pool?

**Reply by scoville07** [reply]
in pool space you can set expected partials. higher value per hour results in less difficulty and high compute used but smoother estimations but although less max farm size. less value results in the opposite. I found that on average it results in the same amount xch you get from the pool. So I go with a lower value of expected partials, because it needs less compute and gives more max farm size

**Reply by JuhaKalevi** [reply]
There is a lower limit of 5 partials per hour and it results in around 1100 difficulty for 669TB of C19 plots. It seems like the required partial difficulty wouldn't be possible there.

**Comment by madMAx43v3r** [comment]
> Will increasing the difficulty / lowering amount of partials negatively influence farm size estimation by my pool?

Yes but it's just a display issue.

**Reply by DudSeb** [reply]
So, in simple words, if i will have 500 Tb of C33 plots at one machine, two 3090ti will manage it ? :) and moving forward if i will have ten machines like this splitted to one big farmer, diff should be set on 200k ? 

Sorry for numb questions, but id like to make it clear before i've start to replotting my 5 Pb farm :D

**Reply by madMAx43v3r** [reply]
> two 3090ti will manage it ?

probably not, need to consider plot filter 256 also. I would recommend C32.

> have ten machines like this splitted to one big farmer, diff should be set on 200k ?

diff should always be like in the chart, no matter how big your farm or number of machines.

**Reply by DudSeb** [reply]
okay than, thank you very much Max, i ll be reporting how it s going i have some cards left after eth mining, so i will put some results here and also add to google document.

**Reply by scoville07** [reply]
would be nice to see, thanks as well.

However after the halving, with my 1.8PB physical capacity, it seems not worth to buy two eg. 4070ti super and replot everything...german price for electricity ruins my bussiness latest when the plot filter will be reduced. unless xch goes up to say 70-80usd...

**Reply by DudSeb** [reply]
@scoville07 how much in total you paying for 1 kwh in Germany ?

**Reply by scoville07** [reply]
I personally pay 32cent, depending on where u live in germany it can go up to 55cent

**Reply by DudSeb** [reply]
I m sitting at 27 cent at average in Poland. Funny thing is that price for kwh is cheaper than distribution cost, clean kwh costs13 cent, while distribution 14 cent. And this prices are without 23 % vat tax. With vat its 0,33 cent. Anyway my way of thinking not changing for years, mine even if u need to pay to this from your pocket. Someday you will be rewarded. I ve remember that i bought so many mining plans from bitdeer when bitcoin dropped under 3k. Everyone was saying it s dead, my wallet now saying something differnet :)) PS. on space farmers you have an option to mine xch but payouts can be directly in btc / eth and few others. I will switch to this for some time.
![coin payout space farmers](https://github.com/madMAx43v3r/chia-gigahorse/assets/121369038/bbab52aa-fc5d-4d47-ad07-9fe1a6295f39)

**Reply by scoville07** [reply]
Yeah, I also will not give up just like that. I have spent so many hours...it is my baby 😁
So price for power seem similar to germany apart from the exact opposite of distribution vs. price for kwh.
However I found I would need insane invest to switch to say C30 or C31. As I am today already at 1,9PB with C18. In a week I have 2PB physical size (so 3,4PB C18). To keep those 2PB and considering plotfilter in June, I would need to split my farm multiple times for C30+, so multiple harvesters with eg. 4070ti etc...and the replotting..so for now I would stick with my two 3060rtx 12gb (not ti), they have a tgp of 175watt or so. So lets see what Max can do with C26-C29..🤪 .
In any case it would be interesting to see how your 4070ti super would perform with c30/c31 if you find the time. It should be more powerful than a rtx a5000??
I saw that option at space farmer, but right now I am transferring xch into other assets right at okx, it is without fee there.
Cheers!

**Comment by XxViper06xX** [comment]
What about multiple 3060Ti for C33?  I have (6) 3060Ti available for use but I don't want to start plotting C33 if it can't be done on even multiple 3060s.

**Reply by madMAx43v3r** [reply]
Yes you can use multiple GPUs, just multiply the estimate for single GPU by the number of GPUs you have.

**Reply by XxViper06xX** [reply]
Discussion in the discord earlier people are under the impression that VRAM doesn't stack so if I'm only using 8GB cards and C33 requires 24GB I can't use multiple cards to achieve 24GB?

**Reply by madMAx43v3r** [reply]
Oh sorry yes, I forgot about that. C32 maximum for 8G VRAM.

**Reply by DudSeb** [reply]
Max i know that its not this topic, but im making now plots to test, and i would like to ask about number of streams, do you have any advices how many streams to  each model of card ?

**Comment by DudSeb** [comment]
Okay guys, here is first result: 

RTX 4070 Ti, k32_v3, diff: 50,000 ,compression level 31:

Max Farm Size @512: 0.795918 PiB (physical)
Max Farm Size @256: 0.397959 PiB (physical)
Max Farm Size @128: 0.198979 PiB (physical) 

I ve mistaken, i have 4070ti and 4070 super. Do not have 4070ti super ;(

**Reply by scoville07** [reply]
Awesome...thanks m8!

Would you mind doing c30 as well? Highly appreciated.

**Reply by DudSeb** [reply]
Yes will do, probably tomorrow :)

**Comment by DudSeb** [comment]
RTX 4070Ti, k32_V3, diff: 10,0000, compression level 32:

 Max Farm Size @512: 0.391024 PiB (physical)
Max Farm Size @256: 0.195512 PiB (physical)
Max Farm Size @128: 0.0977559 PiB (physical)

**Reply by scoville07** [reply]
Awesome! Thanks!

**Comment by DudSeb** [comment]
RTX 4090, k32_V3, diff: 10,0000, compression level 32:

Max Farm Size @512: 0.685709 PiB (physical)
Max Farm Size @256: 0.342854 PiB (physical)
Max Farm Size @128: 0.171427 PiB (physical)

**Reply by scoville07** [reply]
Fabolous! Thanks for all the values!

**Reply by DudSeb** [reply]
I will put more, i have more cards just need find a time :)

**Comment by FoodIsGood** [comment]
Dear Madmax, you state "Partial difficulty is now even more important for maximum farm size. Minimum recommended partial difficulty is 10k for the new 3.0 format."
How to set partial difficulty in space pool?

**Reply by madMAx43v3r** [reply]
It's not possible yet, all you can do is set it as high as they allow based on your farm size.
